crypto/crypto.ts

```diff
@@ -294,11 +294,26 @@ const stdCrypto: StdCrypto = ((x) => x)({
   },
 });
 
-/** FNV (Fowler/Noll/Vo) algorithms names. */
+/**
+ * A FNV (Fowler/Noll/Vo) digest algorithm name supported by std/crypto.
+ *
+ * @deprecated (will be removed in 0.224.0)
```

Comment:
I'd suggest we remove this at version 1.0.0.

Reply:
Gotcha. For all three of them, right?